WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2284,3 +2284,22 @@ WhiteBlackListViewRowCell                  = font-weight:normal;
 WhiteBlackListRemoveItemCell               = width:@BigIconWidth@; @ListItemIconWrapper@
 WhiteBlackListViewItemsNumberUsed          = margin-top:@WhiteBlackViewInnerSpacing@;
 
+######################################
+#   OptionsItems ListView
+######################################
+
+OptionsItemsListViewItemsInnerSpacing      = 10px;
+OptionsItemsListColumnHeaderWrapper        = border-left:0px;
+OptionsItemsListColumnHeader               = padding-left:@OptionsItemsListViewItemsInnerSpacing@;
+OptionsItemsListRow                        = overflow:hidden;
+OptionsItemsListLastRow                    = border-bottom:none;
+OptionsItemsListRows                       = overflow:visible;
+OptionsItemsListRowCell                    = padding-left:@OptionsItemsListViewItemsInnerSpacing@; text-align:left;
+OptionsItemsListRowCellSelected            = @OptionsItemsListRowCell@; line-height:1.7rem; white-space:normal;
```

Comment:
OptionsItemsListRowCell is already going to be applied for normal cells, why we need to reapply in case of selection?